content/docs/buildpack-author-guide/create-buildpack/packaging.md

```diff
@@ -0,0 +1,32 @@
++++
+title="Packaging your buildpack for distribution"
+weight=408
+creatordisplayname = "Natalie Arellano"
```

Comment:
We've stepped away from adding this metadata.

Reply:
Should I remove the entire metadata block? Or just specific fields... this data appears to be present in all of the sibling files.